libraries/Microsoft.Bot.Builder/Inspection/InspectionActivityExtensions.cs

```diff
@@ -33,5 +34,10 @@ public static Activity TraceActivity(this Exception exception)
         {
             return (Activity)Activity.CreateTraceActivity("TurnError", "https://www.botframework.com/schemas/error", exception.Message, "Turn Error");
         }
+
+        public static Activity CloneTraceActivity(this Activity activity) 
+        {
+            return (Activity)Activity.CreateTraceActivity(activity.Name, activity.ValueType, activity.Value, activity.Label);
```

Comment:
>Value [](start = 102, length = 5)

This doesn't clone the value. Did you intend to?
There is a helper function ObjectPath.Clone(activity) which will do this